### ISSUE TITLE
Parsing error: Unknown type:SvelteBoundary

### ISSUE THREAD

**Issue by mustafa0x** [sveltejs/eslint-plugin-svelte#968]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.17.0

### What version of `eslint-plugin-svelte` are you using?

2.46.1

### What did you do?

<details>
<summary>Configuration</summary>

```
import js from '@eslint/js'
import svelte from 'eslint-plugin-svelte'

/** @type {import('eslint').Linter.Config[]} */
export default [
    js.configs.recommended,
    ...svelte.configs['flat/recommended'],
]
```
</details>

```svelte
<svelte:boundary></svelte:boundary>
```

### What did you expect to happen?

work

### What actually happened?

Parsing error: Unknown type:SvelteBoundary

### Link to **GitHub Repo** with Minimal Reproducible Example

tbd

**Comment by baseballyama** [comment]
resolved by https://github.com/sveltejs/eslint-plugin-svelte/pull/956

Please use [3.0.0-next.5](https://github.com/sveltejs/eslint-plugin-svelte/releases/tag/eslint-plugin-svelte%403.0.0-next.5)

**Comment by mustafa0x** [comment]
thanks @baseballyama
